rrmngmnt/storage.py

```diff
@@ -0,0 +1,127 @@
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'vers=3']
+        :type opts: list
+        :return: Path to mount point if succeeded, None otherwise
+        :rtype: str
+        """
+        target = '/tmp/mnt_point' if target is None else target
+        cmd = ['mkdir', target]
+        self.logger.info(
+            "Creating directory %s to use as a mount point", target
+        )
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to create directory as a mount point for. "
+                "Output: %s, Error: %s ",
+                out, err
+            )
+            return None
+
+        cmd = ['mount', source, target]
+        if opts:
+            cmd.extend(opts)
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to mount source %s to target %s. Output: %s",
+                source, target, out
+            )
+            return None
+        return target
+
+    def umount(self, mount_point, force=True, remove_mount_point=True):
+        """
+        Performs 'umount' on 'mount_point' directory and remove
+        the 'mount_point'
+
+        __author__ = "ratamir"
+        :param mount_point: Path to directory that should be un mounted
+        :type mount_point: str
+        :param force: True if force an unmount is required (in case of an
+        unreachable NFS system)
+        :type force: bool
+        :param remove_mount_point: True if mount point should be deleted
+        after 'umount' execution, False otherwise
+        :type remove_mount_point: bool
+        :return: True if umount operation and mount point removal
+        succeeded, False otherwise
+        :rtype: bool
+        """
+        cmd = ['umount', mount_point, '-v']
+        if force:
+            cmd.append('-f')
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "failed to umount directory: %s, output: %s, error: %s",
+                mount_point, out, err
+            )
+            return False
+        if remove_mount_point:
+            rc = self.host.fs.rmdir(mount_point)
+            if not rc:
+                self.logger.error("failed to remove directory %s", mount_point)
+                return False
+        return True
+
+
+class LVMService(Service):
+    """
+    Storage management class to maintain LVM services
+    """
+    def lv_change(self, vg_name, lv_name, activate=True):
+        """
+        Set the LV attribute 'active' (active or inactive)
+
+        __author__ = "ratamir"
+        :param vg_name: The name of the Volume group under which LV is
+        contained
+        :type vg_name: str
+        :param lv_name: The name of the logical volume which needs to be
+        activated or deactivated
+        :type lv_name: str
+        :param activate: True in case the lv change command should activate
```

Comment:
True when the logical volume should be activated, False when it should be deactivated